index.html

```diff
@@ -752,6 +752,18 @@ <h3>Validate Algorithm</h3>
 as indicated in the `statusMessages` array.
         </li>
       </ol>
+
+      <p>
+When a `statusListCredential` URL is dereferenced, server implementations MAY provide a
+mechanism to dereference the status list as of a particular point in time. If such a
+feature is supported, and if query parameters are supported by the URL scheme,
+then the name of the query parameter MUST be `timestamp` and the value MUST be
+a valid URL-encoded [[XMLSCHEMA11-2]] dateTimeStamp string value. The result of
+dereferencing such a timestamped URL MUST be a status list credential containing
+either the status list as it existed at the given point in time, or a 
+<a href="#STATUS_RETRIEVAL_ERROR">STATUS_RETRIEVAL_ERROR</a>.
```

Comment:
I would consider adding a sentence that says something like...

"Providing such a mechanism to dereference the status list as of a particular point in time lends itself to allowing for the functionality of determining as precise as possible timestamps of status changes, at the issuers discretion"

Reply:
Done in 804df0eb5df0692e52165155acac09e17607dcba.